src/main/java/edu/phema/elm_to_omop/api/CohortService.java

```diff
@@ -45,6 +46,12 @@ public CohortService(Config config) throws CohortServiceException {
 
         this.config = config;
         this.valuesetService = new SpreadsheetValuesetService(omopService, config.getVsFileName(), config.getTab());
+        if (config.isTabSpecified()) {
```

Comment:
Not for this PR, but we should probably add some tests at some point for using a spreadsheet with tabs.